sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -228,6 +228,28 @@ class ExplainSuite extends ExplainSuiteHelper with DisableAdaptiveExecutionSuite
     }
   }
 
+  test("SPARK-33853: explain codegen - check presence of subquery") {
+    withTable("df1", "df2") {
+      withSQLConf(SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key -> "true",
+        SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "false") {
```

Comment:
Could you test both `true` and `false` of AQE, please?

Reply:
Ah, I noticed `DisableAdaptiveExecutionSuite`. I'll just remove `SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "false"`.